app/src/components/market/common/transaction_details_card/index.tsx

```diff
@@ -2,8 +2,19 @@ import React from 'react'
 import styled from 'styled-components'
 
 const Wrapper = styled.div`
-  border: 1px solid ${props => props.theme.borders.borderColor};
+  border: 1px solid ${props => props.theme.borders.borderDisabled};
+  background-color: ${props => props.theme.colors.mainBodyBackground};
   padding: 21px;
+  position: relaive;
```

Comment:
Typo - should be `position: relative;`

Reply:
That's not necessary, so let me remove that line.